cmd/internal/cli/inspect.go

```diff
@@ -12,20 +12,22 @@ import (
 	"io"
```

Comment:
In general, I don't think we can rename this file to change the build tag since there are still paths that launch the singularity runtime which wont work on darwin

Reply:
How about I'll split the `inspect.go` to `inspect_darwin.go`, and `inspect_linux.go`. That way we will still be able to inspect a container on darwin (only a sandbox or image with label partition).

Reply:
Sure, but i recommend doing that in a separate pr to keep this diff from getting even larger

Reply:
I see its already there, so nevermind :+1: